pkg/validation/commands_test.go

```diff
@@ -195,29 +203,66 @@ func TestValidateCommandComponent(t *testing.T) {
 	}{
 		{
 			name:    "Valid Exec Command",
-			command: generateDummyExecCommand("command", component, &v1alpha2.CommandGroup{Kind: runGroup}),
+			command: generateDummyExecCommand("command", containerComponent, &v1alpha2.CommandGroup{Kind: runGroup}),
 		},
 		{
 			name:    "Invalid Exec Command with missing component",
 			command: generateDummyExecCommand("command", "", &v1alpha2.CommandGroup{Kind: runGroup}),
 			wantErr: &invalidCmdErr,
 		},
 		{
-			name:    "Valid Exec Command with invalid component",
-			command: generateDummyExecCommand("command", invalidComponent, &v1alpha2.CommandGroup{Kind: runGroup}),
+			name:    "Valid Exec Command with non-exist component",
```

Comment:
probably remove the world "Valid" if it references wrong component, same for the others